pkg/policy/distillery_test.go

```diff
@@ -1127,3 +1160,167 @@ func Test_AllowAll(t *testing.T) {
+			},
+		}}).
+		WithEgressDenyRules([]api.EgressDenyRule{{
+			EgressCommonRule: api.EgressCommonRule{
+				ToEntities: api.EntitySlice{api.EntityWorld},
+			},
+		}}).
+		WithEndpointSelector(api.WildcardEndpointSelector)
+	cpyRule                       = *ruleL3L4__DenyWorld
+	ruleL3L4__DenyWorldWithLabels = (&cpyRule).WithLabels(labels.LabelWorld.LabelArray())
+	mapKeyL3L4__WorldIngress      = Key{identity.ReservedIdentityWorld.Uint32(), 0, 0, trafficdirection.Ingress.Uint8()}
+	mapKeyL3L4__WorldEgress       = Key{identity.ReservedIdentityWorld.Uint32(), 0, 0, trafficdirection.Egress.Uint8()}
+	mapEntryL3L4__Deny            = MapStateEntry{
+		ProxyPort:        0,
+		DerivedFromRules: labels.LabelArrayList{nil},
+		IsDeny:           true,
+		owners:           map[MapStateOwner]struct{}{},
+	}
+	mapEntryL3L4__Allow = MapStateEntry{
+		ProxyPort:        0,
+		DerivedFromRules: labels.LabelArrayList{nil},
+		owners:           map[MapStateOwner]struct{}{},
+	}
+	worldLabelArrayList     = labels.LabelArrayList{nil, labels.LabelWorld.LabelArray()}
+	mapEntryL3L4__WorldDeny = MapStateEntry{
+		ProxyPort:        0,
+		DerivedFromRules: labels.LabelArrayList{nil},
+		IsDeny:           true,
+		owners:           map[MapStateOwner]struct{}{},
+	}
+	mapEntryL3L4__WorldDenyWithLabels = MapStateEntry{
+		ProxyPort:        0,
+		DerivedFromRules: worldLabelArrayList,
+		IsDeny:           true,
+		owners:           map[MapStateOwner]struct{}{},
+	}
+
+	worldIPIdentity        = identity.NumericIdentity(16324)
+	worldCIDR              = api.CIDR("192.0.2.3/32")
+	lblWorldIP             = labels.ParseSelectLabelArray(fmt.Sprintf("%s:%s", labels.LabelSourceCIDR, worldCIDR))
+	ruleL3L4__AllowWorldIP = api.NewRule().
+				WithIngressRules([]api.IngressRule{{
+			IngressCommonRule: api.IngressCommonRule{
+				FromCIDR: api.CIDRSlice{worldCIDR},
+			},
+		}}).
+		WithEgressRules([]api.EgressRule{{
+			EgressCommonRule: api.EgressCommonRule{
+				ToCIDR: api.CIDRSlice{worldCIDR},
+			},
+		}}).
+		WithEndpointSelector(api.WildcardEndpointSelector)
+
+	worldSubnetIdentity = identity.NumericIdentity(16325)
+	worldSubnet         = api.CIDR("192.0.2.0/24")
+	worldSubnetRule     = api.CIDRRule{
+		Cidr: worldSubnet,
+	}
+	lblWorldSubnet       = cidr.GetCIDRLabels(netip.MustParsePrefix(string(worldSubnet)))
+	ruleL3L4__DenySubnet = api.NewRule().
+				WithIngressDenyRules([]api.IngressDenyRule{{
+			IngressCommonRule: api.IngressCommonRule{
+				FromCIDRSet: api.CIDRRuleSlice{worldSubnetRule},
+			},
+		}}).
+		WithEgressDenyRules([]api.EgressDenyRule{{
+			EgressCommonRule: api.EgressCommonRule{
+				ToCIDRSet: api.CIDRRuleSlice{worldSubnetRule},
+			},
+		}}).
+		WithEndpointSelector(api.WildcardEndpointSelector)
+	mapKeyL3L4__SubnetIngress = Key{worldSubnetIdentity.Uint32(), 0, 0, trafficdirection.Ingress.Uint8()}
+	mapKeyL3L4__SubnetEgress  = Key{worldSubnetIdentity.Uint32(), 0, 0, trafficdirection.Egress.Uint8()}
+
+	ruleL3L4__DenySmallerSubnet = api.NewRule().
```

Comment:
These variables are named `ruleL3L4*`, but I only see an L3 component. Maybe these should be...?
```suggestion
	ruleL3____DenySmallerSubnet = api.NewRule().
```

Reply:
incidentally just fyi, the reason for all these underscore characters in these variable names in the existing cases is just to allow the table to line up in the test listings below so it's easier to compare the results of the output.